docs/concepts-throttling.md

```diff
@@ -0,0 +1,167 @@
+Per subscription | <li>If it's a non-partitioned topic, the rate limit refers to the maximum number of messages a subscription can deliver to clients per unit of time.</li><li>If the subscribed topic has multiple partitions, the rate limit refers to the maximum number of messages the subscription can deliver per partition per unit of time. In other words, a subscription's actual dispatch rate limit for a [partitioned topic](concepts-messaging.md#partitioned-topics) is N times the configured one (N is the number of partitions inside the topic). For example, the topic `t1` has two partitions with the subscription `s1`. If you set the rate limit to `10/s`, then the rate limit for `s1` in both `t1-p0` and `t1-p1` is `10/s`, while the total rate limit of `s1` is `20/s`.</li>
+
+:::note
+
+The dispatch rate limits configured at multiple levels take effect simultaneously (logical AND).
+
+:::
+
+### Throttling approaches
+
+The following table outlines multiple approaches to configure the dispatch rate limits at different levels.
+
+Approach | Per cluster | Per topic | Per subscription
+:--------|:------------|:----------|:----------------
+Set [broker configurations](#throttling-configurations) or [dynamic broker configurations](admin-api-brokers.md#dynamic-broker-configuration) | <li>`dispatchThrottlingRateInMsg`</li><li>`dispatchThrottlingRateInByte`</li> | <li>`dispatchThrottlingRatePerTopicInMsg`</li><li>`dispatchThrottlingRatePerTopicInByte`</li><br />It applies to all topics in the cluster. | <li>`dispatchThrottlingRatePerSubscriptionInMsg`</li><li>`dispatchThrottlingRatePerSubscriptionInByte`</li><br />It applies to all subscriptions in the cluster.
+Set namespace policies | N/A | Refer to [Configure dispatch throttling for topics](admin-api-namespaces.md#configure-dispatch-throttling-for-topics). | Refer to [Configure dispatch throttling for subscriptions](admin-api-namespaces.md#configure-dispatch-throttling-for-subscription).
+Set topic policies | N/A | Refer to [Set topic-level dispatch rate](https://pulsar.apache.org/admin-rest-api/#operation/persistent_setDispatchRate). | Refer to [Set subscription-level dispatch rate](https://pulsar.apache.org/admin-rest-api/#operation/getSubscriptionLevelDispatchRate).<br />It applies to all subscriptions in a topic.
```

Comment:
Please confirm if these links are in the correct MD format.

Reply:
This link validity has a dependency on https://github.com/apache/pulsar/pull/19193. I'll double-check it after https://github.com/apache/pulsar/pull/19193 is merged.

Reply:
Hi @momo-jun 

Sorry, this PR should be terminated because I have found a better solution and I will submit a new PR next week or later.

Reply:
@poorbarcode Thanks for letting us know. Are you going to develop a new feature? Keep us posted. I'll close this PR when the new one is ready.

Reply:
> Are you going to develop a new feature? Keep us posted. I'll close this PR when the new one is ready.


No. I will just rewrite the PR https://github.com/apache/pulsar/pull/19193.

Reply:
Hi @momo-jun @heesung-sn 

I have updated the changes in apache/pulsar#19193 to solve the admin API link conflict. Please help review it when you have time.

Reply:
Hi @momo-jun 

https://github.com/apache/pulsar/pull/19193 has been merged.